doc/aliases/subpage_info.html

```diff
@@ -0,0 +1,29 @@
+<script type="text/javascript">
+//<![CDATA[
+(function () {
+var $getSpan = $('span.newPageId').last();
+var pageId = $getSpan.html();
+var $addParagraph = $( "<p class='Info'></p>" );
+$addParagraph.insertBefore( $getSpan );
+$.get( pageId+".html").then( function (data) {
+    var $elements = $(data).find("div.newInnerHTML");
+    $elements = alphabeticOrderElements($elements);
+    var seen = {};
+    var str = "Toggle: >";
+    $elements.each(function() {
+       var txt = $(this).html();
+       if (seen[txt]){
```

Comment:
I think it is not a mistake.

The logic was:
- If it was already seen then we don't need to add it to the string anymore.

But I will check it out.